app/services/funnel/doc_auth/register_step.rb

```diff
@@ -10,7 +10,6 @@ class RegisterStep
         back_image
         mobile_front_image
         mobile_back_image
-        selfie
```

Comment:
Do you want to extend the `ignored_columns` array in the [DocAuthLog model](https://github.com/18F/identity-idp/blob/db847b4667f01f51af13ecd752f5a51084ff08f7/app/models/doc_auth_log.rb#L10-L15) for future deletion, or leave that data in place?

Reply:
I think we should leave the doc auth logs along for now